src/api/app/models/user.rb

```diff
@@ -638,13 +638,17 @@ def delete!
     true
   end
 
+  def group_ids
```

Comment:
"Possibly there are other ways"

There are :-) Don't overwrite the group_ids method you get from the [has_and_belongs_to_many](https://github.com/openSUSE/open-buildservice/blob/134e04b7e259987195a9e599927fd1e19e8a18c2/src/api/app/models/user.rb#L54), but keep the changes you've made below.

Reply:
> "Possibly there are other ways"
> 
> There are :-) Don't overwrite the group_ids method you get from the [has_and_belongs_to_many](https://github.com/openSUSE/open-buildservice/blob/134e04b7e259987195a9e599927fd1e19e8a18c2/src/api/app/models/user.rb#L54), but keep the changes you've made below.

That's still the same way - just removing code duplication.

Reply:
Exactly